src/main/java/core/basesyntax/service/impl/WriteToFileServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriteToFileService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class WriteToFileServiceImpl implements WriteToFileService {
+    private static final String PATH_TO_REPORT = "src/main/resources/report.csv";
+
+    @Override
+    public void writeToFile(String report) {
+        Path path = Paths.get(PATH_TO_REPORT);
+
+        if (!Files.exists(path)) {
+            try {
+                Files.createFile(path);
+            } catch (IOException e) {
+                throw new RuntimeException("File can't be created.", e);

```

Comment:
I think that this check is unnecessary, because if there is no file, then it will be created and write everything necessary to the file, and if there is, it will overwrite

Reply:
agree